riva/clients/tts/riva_tts_client.cc

```diff
@@ -47,9 +50,51 @@ DEFINE_string(
     zero_shot_audio_prompt, "",
     "Input audio file for Zero Shot Model. Audio length between 0-3 seconds.");
 DEFINE_int32(zero_shot_quality, 20, "Required quality of output audio, ranges between 1-40.");
+DEFINE_string(user_dictionary, "", " User dictionary containing graph-to-phone custom words");
 
 static const std::string LC_enUS = "en-US";
 
+std::string 
+ReadUserDictionaryFile(const std::string& dictionary_file)
+{
+  std::string dictionary_string;
+  if (!dictionary_file.empty()) {
+    std::ifstream infile(dictionary_file);
+
+    if (infile.is_open()) {
+      std::string line;
+      bool is_first_line = true;
+
+      while (std::getline(infile, line)) {
+        if (is_first_line) {
+          // Skip the first line it may contains headers
+          is_first_line = false;
```

Comment:
first line may also contain valid data, right?

Reply:
would it be ok to remove the first line skip part? we can assume the file would be properly set by the user, as we would be mentioning the right format for this file in docs. In wrong file is provided still, user will have unexpected results.

Reply:
Removed as suggested

Reply:
need to remove from riva_tts_perf_client also

Reply:
looks good